### PR TITLE
Pulssi 688

### PR DESCRIPTION
**Pull request by kompa3**

### CODE REVIEW

**Comment by kompa3** [comment]
I didn't yet create a release in Github. Waiting for PR merge to master, then I'll add a tag and upload the rpm files to Github releases.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+chmod 0644 /etc/zabbix/zabbix-agentd.d/SCRIPTNAME.conf
+```
+
+# How to Release a New Version (for Digia Pulssi Developers)
+
+First build the package creation containers:
+
+docker build -t zabbix-rpm:centos5 -f Dockerfile.centos5 .
+docker build -t zabbix-rpm:centos6 -f Dockerfile.centos6 .
+docker build -t zabbix-rpm:centos7 -f Dockerfile.centos7 .
```
Comment (sapajune) [inline README.md]: Missing code section markup.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+Then run the following commands to produce new installation packages for different platforms:
+
+docker run --rm -v $(pwd)/RPMS:/usr/src/redhat/RPMS zabbix-rpm:centos5
+docker run --rm -v $(pwd)/RPMS:/root/rpmbuild/RPMS zabbix-rpm:centos6
+docker run --rm -v $(pwd)/RPMS:/root/rpmbuild/RPMS zabbix-rpm:centos7
```
Comment (sapajune) [inline README.md]: Missing code section markup.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+- Configuration file (something.conf): /etc/zabbix/zabbix-agentd.d/
+
+Modify the file permissions as follows (the owner should be root):
+
+```
+chmod 0755 /etc/zabbix/scripts/SCRIPTNAME
+chmod 0644 /etc/zabbix/zabbix-agentd.d/SCRIPTNAME.conf
+```
+
+# How to Release a New Version (for Digia Pulssi Developers)
```
Comment (sapajune) [inline README.md]: Could this be a script?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+Then run the following commands to produce new installation packages for different platforms:
+
+docker run --rm -v $(pwd)/RPMS:/usr/src/redhat/RPMS zabbix-rpm:centos5
+docker run --rm -v $(pwd)/RPMS:/root/rpmbuild/RPMS zabbix-rpm:centos6
+docker run --rm -v $(pwd)/RPMS:/root/rpmbuild/RPMS zabbix-rpm:centos7
+
+Finally, copy the zabbix-agent rpm packages from RPMS folder to Github releases section.
+
+- Note that CentOS 5 package name must be modified to include "el5" tag similarly to the others
+- Remove "centos" from CentOS 7 package name
```
Comment (sapajune) [inline README.md]: How do we do version numbers for releases if we do multiple releases with same zabbix version?

Comment (kompa3) [inline README.md]: Good point. Added one version number after Zabbix Agent release version for this purpose.

File: README.md (hunk 3 above)
Comment (kompa3) [inline README.md]: Yes, fixed

**Comment by kompa3** [comment]
Yep, added URL_ZABBIX_SRPM and clarified the versioning instructions a little.

**Pull request merged**